meetup/src/main.rs

```diff
@@ -181,7 +43,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         )
         .get_matches();
 
-    let scan_interval = matches.value_of("publisher-scan-interval").unwrap();
+    matches.value_of("publisher-scan-interval").unwrap();
```

Comment:
is this actually not used anywhere?

Reply:
It was being used for the `remove_expired_publishers`. I'll add it back in as a struct property for now.

Reply:
it looks like this line might not be needed any more. I think things are pretty traceable from the option name.